Examples/SPECAN_TXP_LOADPULL/SPECAN_TXP_LOADPULL.cs

```diff
@@ -0,0 +1,173 @@
+﻿using NationalInstruments.ModularInstruments.NIRfsg;
+using NationalInstruments.ReferenceDesignLibraries.FocusITuner;
+using NationalInstruments.RFmx.InstrMX;
+using NationalInstruments.RFmx.SpecAnMX;
+using System;
+using System.Collections.Generic;
+using static NationalInstruments.ReferenceDesignLibraries.SG;
+
+namespace NationalInstruments.ReferenceDesignLibraries.Examples
+{
+    public class SPECAN_TXP_LOADPULL
+    {
+        #region Declaring Parameters
+        //shared Parameters
+        public string resourceName;
+        public double centerFrequency;
+        public string signalStringSpecan;
+        public string resultStringSpecan;
+
+        //Tuner Configration
+        public string tunerAddress;
+        public FocusTuner.CommonConfiguration commonConfiguration;
+        Complex[] gammaSweep;
+
+        //Waveform
+        public string filePath;
+
+        //Generator Configuration
+        SG.InstrumentConfiguration sgInstrConfig;
+
+        //Analyzer Configuration 
+        public SA.RFmxInstr.InstrumentConfiguration saInstrConfig;
+        public SA.CommonConfiguration saCommonConfig;
+        public SA.AutoLevelConfiguration saAutolevelConfig;
```

Comment:
any reason you didnt add a using directive for the SA namespace?

Reply:
What does directive mean here?

It was copied from existing example as a template and didn't explicitly remove the namespace.

Reply:
Add the namespace and drop the SA.